docs/index.rst

```diff
@@ -1,16 +1,77 @@
+.. _baseband_docs:
+
 ********
 Baseband
 ********
 
-Welcome to the Baseband documentation! Baseband is a package for reading and 
+Welcome to the Baseband documentation! Baseband is a package for reading and
 writing VLBI and other radio baseband files, with the aim of simplifying and
-streamlining data conversion and standardization.
+streamlining data conversion and standardization.  It provides:
+
+- File input/output functions for supported radio baseband formats, listed under
+  :ref:`specific file formats <specific_file_formats_toc>`.
+- Data container classes for supported formats, which can be used for accessing
+  and manipulating data samples and for generating baseband formats from
+  user-defined data.  The latter functionality can be used for converting
+  between file formats.
+- Classes for reading from and writing to an ordered sequence of files as if
+  it was a single file.
+
+.. _using_baseband_toc:
+
+Basics
+======
+
+.. toctree::
+   :maxdepth: 1
+
+   install
+   tutorials/getting_started
+   tutorials/glossary
+   authors_for_sphinx
+
+.. _specific_file_formats_toc:
+
+Specific file formats
+=====================
+
+Baseband's code is subdivided into its supported file formats, and the
+following sections contain format specifications, usage notes,
+troubleshooting help and APIs for each.
 
 .. toctree::
-  :maxdepth: 2
+   :maxdepth: 1
 
-  baseband/install
+   vdif/index
+   mark5b/index
+   mark4/index
+   dada/index
+   gsb/index
 
-  baseband/index
+.. _core_utilities_toc:
+
+Core framework and utilities
+============================
+
+These sections contain APIs and usage notes for the sequential file opener, and
+the API for the set of core utility functions and classes located in
+:mod:`~baseband.vlbi_base`.
+
+.. toctree::
+   :maxdepth: 1
+
+   helpers/index
+   vlbi_base/index
+
+.. _dev_docs_toc:
+
+Developer documentation
+=======================
+
+The developer documentation feature tutorials for supporting new formats or
+format extensions such as VDIF EDV.
+
+.. toctree::
+   :maxdepth: 1
 
-  authors_for_sphinx
+   tutorials/new_edv
```

Comment:
Not for this PR, but for my #119, but do you want the build status and citation information in the main documentation too, a la the [Gala docs](https://gala.readthedocs.io/en/latest/)?  They're currently only in the master README.rst.

Reply:
Let's discuss there...